src/presign/participant.rs

```diff
@@ -993,7 +993,7 @@ impl PresignParticipant {
             .local_storage
             .retrieve::<storage::RoundOnePublicBroadcast>(message.from())?;
         let public = RoundThreePublic::try_from(message)?;
-        public.verify(
+        public.clone().verify(
```

Comment:
note: This seemed weird to me and I think we should change this API at some point, but for now I [made a comment](https://github.com/boltlabs-inc/tss-ecdsa/issues/352#issuecomment-1577346861) on the most relevant issue.